src/Xamarin.Android.Build.Tasks/Tasks/Aapt2.cs

```diff
@@ -0,0 +1,169 @@
+﻿// Copyright (C) 2011 Xamarin, Inc. All rights reserved.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Xml;
+using System.Xml.Linq;
+using Microsoft.Build.Utilities;
+using Microsoft.Build.Framework;
+using System.Text.RegularExpressions;
+using System.Collections.Generic;
+using Xamarin.Android.Tools;
+using ThreadingTasks = System.Threading.Tasks;
+
+namespace Xamarin.Android.Tasks {
+	
+	public class Aapt2 : AsyncTask {
+
+		protected Dictionary<string, string> resource_name_case_map = new Dictionary<string, string> ();
+
+		protected struct OutputLine {
```

Comment:
Should `OutputLine` and associated "plumbing" be moved into `AsyncTask` (or some other intermediate base type) so it can be shared with `Aapt.cs`?